holoviews/tests/test_annotators.py

```diff
@@ -25,7 +25,7 @@ def test_compose_annotators(self):
         self.assertEqual(len(tables), 3)
 
     def test_annotate_overlay(self):
-        layout = annotate(Wikipedia() * Points([]), annotations=['Label'])
+        layout = annotate(OSM() * Points([]), annotations=['Label'])
```

Comment:
Is it possible to use a tile service from Esri or another company? I feel running the tests pulling data from the nice people of OSM.

Reply:
Definitely, the only reason I switched to OSM was because I got this warning in a notebook when running `Wikipedia()`:
`WARNING:param.main: Wikipedia tile source no longer available outside wikimedia domain as of April 2021; switching to OpenStreetMap (OSM) tile source. See release notes for HoloViews 1.14.4 for more details`

Though I'm not sure that date is actually being pulled when running the test, but I could be wrong. 

I will change it to `EsriStreet`.